src/libsystemd/sd-id128/id128-util.c

```diff
@@ -54,7 +55,7 @@ int id128_read_fd(int fd, Id128FormatFlag f, sd_id128_t *ret) {
          * This returns the following:
          *     -ENOMEDIUM: an empty string,
          *     -ENOPKG:    "uninitialized" or "uninitialized\n",
-         *     -EINVAL:    other invalid strings. */
+         *     -EIO:       other invalid strings. */
```

Comment:
I know this is very close to bike-shedding, but I think we shouldn't use EIO. EIO is for "input/output error", i.e. the error you'd get if the disk or file system is borked, and this error code could be returned by the underlying read/write functions. It'd be better to use something unambiguous, like EUCLEAN.